integration-tests/pom.xml

```diff
@@ -35,4 +35,12 @@
         <module>kyuubi-flink-it</module>
     </modules>
 
+    <profiles>
+        <profile>
+            <id>kubernetes</id>
```

Comment:
I'm wondering if we can use a profile it to activate all integration tests moudles? The `kubernetes` is quite confusing to me.

Reply:
We need this, since the k8s test env is different from others that use minikube action to run the k8s-it.

Reply:
How about rename it to `it-kubernetes` or `it-k8s`?

Reply:
> I'm wondering if we can use a profile it to activate all integration tests moudles? The `kubernetes` is quite confusing to me.

+1 about the concern of naming

Reply:
how about `kubernetes-it` ? since the module names are `kyuubi-kubernetes-it` and `kyuubi-flink-it` , and personally I don't like the short name ..

Reply:
ah I see the `kubernetes-deployment-it` of @yaooqinn , looks fine. It looks clear if we add spark/flink kubernetes it in future.